documentation/design-docs/dotnet-tools.md

```diff
@@ -656,13 +657,43 @@ COLLECT
       -v, --verbose
           Turns on logging for gcdump
 
+      --std-out
+          Also writes plaintext results into stdout.
+
 Examples:
 
     $ dotnet gcdump collect --process-id 1902
     Writing gcdump to file ./20190226_135837_1902.gcdump
     Wrote 12576 bytes to file
     Complete
 
+PRINT
+
+    dotnet-gcdump print -f|--file <input_dump_path>
+    
+    Prints a previously collected gcdump into the stdout
+    
+    Usage:
+      dotnet-gcdump print [options]
+    
+    Options:
+      -f, --file <f>    The file to read gcdump from.
+
+Examples:
+
+    $ dotnet gcdump print -f 20200207_094403_19847.gcdump | head -10
+     1,335,288  (Dump size)
+       299,845  (Total allocations)
```

Comment:
Allocations could be a misinterpreted as meaning the total number of allocations the application has done as opposed to the number of allocated objects which are still surviving in the GC heap at this point. Anyone who pauses to think about it will probably figure out the correct meaning, but we can avoid the issue by saying 'objects' instead.